src/utils/error.ts

```diff
@@ -338,10 +338,28 @@ export class AppErrorCodes {
   public static UNABLE_TO_PARSE_RESPONSE = 'unable-to-parse-response';
 }
 
+/**
+ * Base class for client error codes and their default messages.
+ */
+export class BaseClientErrorCode {
```

Comment:
This is a fairly weak abstraction. It's a class with no specific shape, but a few static members. I think this ought to be an interface:

```typescript
interface ErrorCodeConfig {
  invalidArg: string;
  invalidCredential: string;
  internalError: string;
}
```

WDYT?

Reply:
Agreed! Given the number of error codes needed in the token verifier util this makes sense. Thanks!